### PR TITLE
improving sequential widgets disable on old UI disable and reverse + forcing default keybinds for old UI

### PR DESCRIPTION
**Pull request by PepeAmpere**
[https://trello.com/c/PodkNXEb/]

### CODE REVIEW

File: LuaUI/Widgets/gui_s44_enableOldUI.lua [hunk 1]
```diff
@@ -10,8 +10,27 @@ function widget:GetInfo()
 	}
 end
 
+-- syntax + defaults from https://github.com/spring/spring/blob/104.0/doc/uikeys.txt
+-- complex syntax: https://springrts.com/wiki/Uikeys.txt
+
 local MAP_HEIGHT_RATIO = 0.12
 
+local defaultKeyBinds = {
```
Comment (sanguinariojoe) [inline LuaUI/Widgets/gui_s44_enableOldUI.lua]: Are you overriding custom keybinds? (https://springrts.com/wiki/Uikeys.txt).

Actually I don't know if this issue can be worked around... But that will be in fact a really bad business if someone ask for help (I'm afraid not much people will be aware of that)

Comment (PepeAmpere) [reply LuaUI/Widgets/gui_s44_enableOldUI.lua]: ℹ️ We are overriding custome keybinds for oldUI for given explicit list of commands

* `F1` - showElevation
* `F2` - showPathTraversability
* `F3` - lastMsgPos
* `F4` - showMetalMap
* `F5` - hideinterface
* `F6` - noSound
* `Ctrl+F1` - viewfps
* `Ctrl+F2` - viewta
* `Ctrl+F3` - viewtw
* `Ctrl+F4` - viewrot
* `Shift+Esc` - quitmenu
* `Any+0xa7` drawinmap and `Any+;` drawinmap

any other custom settings are not affected.

----

ℹ️ In new UI there is option to reddefine the keys via ingame menu and it is not affected by this widget
ℹ️ Also in new UI you can reset settings to "default Spring keybinds" via ingame menu

----

ℹ️ in hypotetical situation user wants to override those few defaults for oldUI mentioned above, he can still write a custom widget to do that thing for him. I think that's acceptable solution.

----

Yes, I think there is no cheap/easy way to worked-around this issue. We cannot be flexible and provide reliable setting in two UI systems at the same time unless we spend much more time on that. 

On the other hand I would not see that pesimistic - the suggested user path is to use new UI where you can set all keybinds via interface - that system works and user can be as flexible as he wants. We also fixed the oldUI environment/setup quite a lot so oldUI users can still play under reasonable setup without newUI system depressing the unnecessarily. 😃

Comment (sanguinariojoe) [reply LuaUI/Widgets/gui_s44_enableOldUI.lua]: Not ideal, but fair enough for me!

**Comment by PepeAmpere** [comment]
Waiting for second review. If no issue found, we get it to baseline. (ℹ️ rebase&merge)

**Comment by PepeAmpere** [comment]
Merged but seems the github solution creates unnecessary new commit during the rebase. I wonder if I hit the proper procedure I do personally in my SourceTree one day :D. Btw it looks like users are able to edit the git commands in git command line used by this provider. Thats one way to go next time.

**Pull request merged**